### DISCUSSION TITLE
How do you merge a pull request without review?

### DISCUSSION THREAD

**Discussion by jaimeken127** [community/community#19203] [Pull Requests] [answered]
I've done pull requests before but I'm not sure how to designate a collaborator as a reviewer. And yes, this is for the badge :)

**Comment by justin-2028** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/96811261/175119857-98259b36-e882-4cad-aa39-27de4c2826c0.png)

Follow this guide in the image above. The right sidebar of the PR tab is easy to ignore so that might be the reason for your confusion. Hope you can get the YOLO badge 👍 

Ask me if you have any further questions

**Reply by jaimeken127** [reply]
OH yeah I overlooked the right sidebar

**Reply by jaimeken127** [reply]
thanks and I'll ask if there's any further issues

**Reply by pal404error** [reply]
for yolo badge user must give review or 
Ignore it ?

**Reply by PaulThomas20002** [reply]
> for yolo badge user must give review or Ignore it ?

without review
